01-creating-a-transaction/index.js

```diff
@@ -7,29 +7,33 @@ const sdk = new SDK({ url: 'https://eos.greymass.com' }) // loaded with default
 
 async function main() {
     const session = await sdk.login('myapp') // login also restores existing session unlike anchor-link, can have option for forcing re-login
-    
+
     // Loads contract by name
     const contract = await sdk.getContract('rewards.gm')
-    
+
     // Work with table data
-    const usersTable = await contract.getTable('users')
-    const row = await usersTable.getByPrimary('teamgreymass')
-    const amount = row.getColumn('amount')
-    
+    const usersTable = contract.getTable('users')
+    const rows = await usersTable.getByPrimary('teamgreymass')
```

Comment:
Fetching by the primary index could return multiple rows - I'm assuming that'd be the default behavior?